### ISSUE TITLE
Change Image resizing to match perfectly with tiktok interface

### ISSUE THREAD

**Issue by o78** [elebumm/RedditVideoMakerBot#645]
**Is your feature request related to a problem? Please describe.**
With the current "minus width" there is text behind the tiktok interface, while this is bigger its better to read having it perfectly aligned next to it. 

**Describe the solution you'd like**
my results were promising when changing the resize values from -100 to -370. (video_creation/final_video.py line 64)

**Describe alternatives you've considered** (optional)

**Additional context** (optional)

**Comment by hakupila** [comment]
i agree to that, it sometimes overlaps with with the tiktok interface
